onnxscript/values.py

```diff
@@ -13,7 +13,7 @@
 import numpy as np
 import onnx
 
-from onnxscript import debuginfo, irbuilder, tensor
+from onnxscript import debuginfo, evaluator, irbuilder, tensor
```

Comment:
Why not import evaluator down below, when needed? It avoids the import cycle warning above, and tells us something about where exactly it is used, which will be useful if we want to do any refactoring to eliminate more of these cycle warnings.

Reply:
Oh, I see, it is for the type annotation: I guess you could enclose `evaluator.Evaluator` in a string? But, anyway, I think importing at top is fine too, except for these cycle warnings

Reply:
I did it follow the general recommendation of importing at the top. having imports in between code generally makes the code harder to manage as we move things around. I will see if there is a way to handle the cycles

Reply:
ok this doesn't work yet because the generated code is importing objects directly.

`from onnxscript.values import Op, Opset` etc.

we may need to clean that up first